include/intersectables/triangle.h

```diff
@@ -12,6 +13,70 @@ class Material;
 // distance is along the direction that the normal points (meaning that the
 // sign of distance matters)
 class Triangle : public Intersectable {
+ private:
+  BoundingBox computeAABB(const Vector3f& a, const Vector3f& b, const Vector3f& c) {
```

Comment:
Independent of all members of this class, so consider to instead move to an anonymous function in the cpp file:

namespace {
  BB computeAABB(..) {
  }
}